pallets/automation-time/src/lib.rs

```diff
@@ -317,14 +318,15 @@ pub mod pallet {
 			+ TypeInfo
 			+ MaxEncodedLen;
 
-		/// Handler for fees
-		type FeeHandler: HandleFees<Self>;
-
 		/// Utility for sending XCM messages
-		type XcmSender: SendXcm;
+		type XcmpTransactor: XcmpTransactor<Self::AccountId, Self::CurrencyId>;
 
-		type Origin: From<<Self as SystemConfig>::Origin>
-			+ Into<Result<CumulusOrigin, <Self as Config>::Origin>>;
+		/// The currencyId for the native currency.
+		#[pallet::constant]
+		type GetNativeCurrencyId: Get<Self::CurrencyId>;
```

Comment:
I didn't see you use this anywhere

Reply:
https://github.com/OAK-Foundation/OAK-blockchain/blob/laura/xcmp-handler-in-automation-pallet/pallets/automation-time/src/lib.rs#L129

Reply:
ugh that damn thing. We need to remove that at some point